packages/dendron-viz/src/process-dir.js

```diff
@@ -0,0 +1,68 @@
+import fs from "fs";
+import * as nodePath from 'path';
+import { shouldExcludePath } from './should-exclude-path';
```

Comment:
why is this a `js` file?

Reply:
removed `process-dir.js` as it is not used